alphacsc/swm.py

```diff
@@ -0,0 +1,166 @@
+"""
+Code adopted from Voytek Lab package neurodsp:
+https://github.com/voytekresearch/neurodsp/blob/master/neurodsp/shape/swm.py
+
+The sliding window matching algorithm identifies the waveform shape of
+neural oscillations using correlations.
+"""
+
+# Authors: Scott Cole
+#          Mainak Jas <mainak.jas@telecom-paristech.fr>
```

Comment:
the neurodsp module is under the MIT license, but if you want to use a different license, I'm fine with that